src/main/java/org/disq_bio/disq/impl/formats/vcf/VcfSink.java

```diff
@@ -90,7 +103,28 @@ public void save(
         out.write(BlockCompressedStreamConstants.EMPTY_GZIP_BLOCK);
       }
     }
+
+    List<String> vcfParts =
+        fileSystemWrapper
+            .listDirectory(jsc.hadoopConfiguration(), tempPartsDirectory)
+            .stream()
+            .filter(new HiddenFileFilter())
+            .collect(Collectors.toList());
+    List<Long> partLengths = new ArrayList<>();
+    for (String part : vcfParts) {
+      partLengths.add(fileSystemWrapper.getFileLength(jsc.hadoopConfiguration(), part));
+    }
+
     new Merger().mergeParts(jsc.hadoopConfiguration(), tempPartsDirectory, path);
+    if (writeTbiFile) {
+      new TbiMerger(fileSystemWrapper)
+          .mergeParts(
+              jsc.hadoopConfiguration(),
+              tempPartsDirectory,
+              path + TabixUtils.STANDARD_INDEX_EXTENSION,
```

Comment:
On the very unlikely chance that `TabixUtils.STANDARD_INDEX_EXTENSION` and `TabixIndexWriteOption.getIndexExtension()` get out of sync, it might be better to use one or the other consistently